src/main/java/org/skyscreamer/jsonassert/comparator/JSONCompareUtil.java

```diff
@@ -59,15 +60,30 @@ private JSONCompareUtil() {
     /**
      * Searches for the unique key of the {@code expected} JSON array.
      *
+     * @param key the current Json path
      * @param expected the array to find the unique key of
+     * @param customizations the list of customizations, if any
      * @return the unique key if there's any, otherwise null
      * @throws JSONException JSON parsing error
      */
-    public static String findUniqueKey(JSONArray expected) throws JSONException {
+    public static String findUniqueKey(String key, JSONArray expected, 
+    		Customization... customizations) throws JSONException {
```

Comment:
This seems to be something that's not used as `varargs`. In that the end-user won't pass a variable number of input parameters in; the framework will use this with a fixed size array. `Customization[]` might be cleaner.